app/views/items/index.html.erb

```diff
@@ -8,7 +8,11 @@
 			<div class="item_card index_item_<%= item.id %>">
 				<img class="item-card-img" src="<%= item.image %>" alt="">
 				<h3><%= link_to item.title, item_path(item) %></h3>
-				<p><%= link_to "Add to Cart", cart_path(item_id: item.id), method: :post, class: "cart btn btn-outline-info" %></p>
+				<% if item.active? %>
+					<%= link_to "Add to Cart", cart_path(item_id: item.id), method: :post, class: "cart-button btn btn-outline-info" %>
```

Comment:
I think this conditional should do us just fine.

Reply:
i concur